src/components/vm/disks/vmDiskColumns.jsx

```diff
@@ -105,3 +115,81 @@ DiskExtras.propTypes = {
     errorPolicy: PropTypes.string,
     idPrefix: PropTypes.string.isRequired,
 };
+
+export const DiskActions = ({ vm, vms, disk, supportedDiskBusTypes, idPrefixRow }) => {
+    const [isActionOpen, setIsActionOpen] = useState(false);
+
+    const Dialogs = useDialogs();
+
+    const onRemoveDisk = () => {
+        return domainDetachDisk({ connectionName: vm.connectionName, id: vm.id, name: vm.name, target: disk.target, live: vm.state === 'running', persistent: vm.persistent })
+                .then(() => domainGet({ connectionName: vm.connectionName, id:vm.id }));
```

Comment:
missing space between `id:` and `vm.id` :)

Reply:
Fixed